components/sup/src/manager/file_watcher.rs

```diff
@@ -3038,6 +3072,24 @@ mod tests {
             }
         }
 
+        //  For the poll watcher tehre are more than one Debounced event received
+        //  and this test will fail.  Instead we can look at the last event and
+        //  ensure that it is correct, as it will be the last entry that will
+        //  determine the final state of the watched.
```

Comment:
should we have both tests just look at the last one if that is the only one that matters?

Reply:
I did not want to change the original test case but will give it a try.